administration/src/bp-modules/applications/JsonFieldElemental.tsx

```diff
@@ -96,7 +97,12 @@ const JsonFieldElemental = ({
     case 'String':
       return (
         <p>
-          {t(getTranslationKey())}: {jsonField.value}
+          {t(getTranslationKey())}:
+          {isEmailValid(jsonField.value) ? (
+            <a href={`mailto:${jsonField.value}`}>{jsonField.value}</a>
```

Comment:
```suggestion
            <a href={`mailto:${jsonField.value}`}> {jsonField.value}</a>
```
i think there is an empty space missing after the label and before the mail address

Reply:
I noticed it but I kept the same format as possible...